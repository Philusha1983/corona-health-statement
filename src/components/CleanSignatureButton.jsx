import React from 'react';
import { Button } from 'react-bootstrap';
import {
  X,
} from 'react-bootstrap-icons';

const CleanSignatureButton = (props) => {

  return (
    <Button variant="outline-dark" onClick={()=> props.onClick()}>
      <X />
      {' '}
      נקה חתימה
    </Button>
  );
}

export default CleanSignatureButton;