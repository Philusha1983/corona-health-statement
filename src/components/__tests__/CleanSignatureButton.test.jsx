import React from 'react';
import { render } from '@testing-library/react';
import CleanSignatureButton from '../CleanSignatureButton';

test('Expect CleanSignatureForm to be rendered', () => {
  const uiElement = render(<CleanSignatureButton />);
  expect(uiElement).toMatchSnapshot();
});