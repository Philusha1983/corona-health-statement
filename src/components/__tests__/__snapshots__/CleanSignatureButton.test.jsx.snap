// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect CleanSignatureForm to be rendered 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <button
        class="btn btn-outline-dark"
        type="button"
      >
        <svg
          fill="currentColor"
          height="1em"
          viewBox="0 0 16 16"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M11.854 4.146a.5.5 0 010 .708l-7 7a.5.5 0 01-.708-.708l7-7a.5.5 0 01.708 0z"
            fill-rule="evenodd"
          />
          <path
            clip-rule="evenodd"
            d="M4.146 4.146a.5.5 0 000 .708l7 7a.5.5 0 00.708-.708l-7-7a.5.5 0 00-.708 0z"
            fill-rule="evenodd"
          />
        </svg>
         
        נקה חתימה
      </button>
    </div>
  </body>,
  "container": <div>
    <button
      class="btn btn-outline-dark"
      type="button"
    >
      <svg
        fill="currentColor"
        height="1em"
        viewBox="0 0 16 16"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clip-rule="evenodd"
          d="M11.854 4.146a.5.5 0 010 .708l-7 7a.5.5 0 01-.708-.708l7-7a.5.5 0 01.708 0z"
          fill-rule="evenodd"
        />
        <path
          clip-rule="evenodd"
          d="M4.146 4.146a.5.5 0 000 .708l7 7a.5.5 0 00.708-.708l-7-7a.5.5 0 00-.708 0z"
          fill-rule="evenodd"
        />
      </svg>
       
      נקה חתימה
    </button>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
